src/main/webapp/assets/feedScript.js

```diff
@@ -77,15 +77,24 @@ document.addEventListener('DOMContentLoaded', onLoad);
 /**
  * Fires as soon as the DOM is loaded.
  */
-function onLoad() {
+async function onLoad() {
+  const collegeid = (new URLSearchParams(window.location.search)).get('collegeid');
```

Comment:
Nit: camelCase for variables + consistency!

Reply:
good catch!